src/sentry/new_migrations/utils.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+
+def drop_column_indexes(schema_editor, model, column_names, use_concurrently=True):
+    """
+    Drops any indexes that use all of the listed columns. Safer than specifying index
```

Comment:
This comment seems ambiguous about what is happening. In your tests, you call this function with `["col_a"]`. The test indexes use "all of the listed columns" (in this case `["col_a"]`) and yet none of them are dropped. Really this will drop an index if its columns are a subset of the listed columns.